common/workunit/workunit.hpp

```diff
@@ -1788,5 +1788,6 @@ extern WORKUNIT_API void executeThorGraph(const char * graphName, IConstWorkUnit
 extern WORKUNIT_API TraceFlags loadTraceFlags(IConstWorkUnit * wu, const std::initializer_list<TraceOption> & y, TraceFlags dft);
 
 extern WORKUNIT_API bool executeGraphOnLingeringThor(IConstWorkUnit &workunit, unsigned wfid, const char *graphName);
+extern WORKUNIT_API bool workunitGraphCacheEnabled;
```

Comment:
I'm not a fan of exported variables.  We tend to provide get()/set() functions everywhere else except for roxie.  I may not care enough to avoid merging though.

Reply:
If you can come up with a good reason, I don't mind changing. But there are plenty of other exported variables around...

Reply:
There are very few, except for roxie.  (E.g. I don't think there are any in this file.)
It generates slightly worse code (an extra indirect) - not significant in this case, but in general worth avoiding.  (Thread local access code is much worse...)